### ISSUE TITLE
gmetric4j license issue

### ISSUE THREAD

**Issue by srdo** [micrometer-metrics/micrometer#1354]
gmetric4j depends on org.acplt.remotetea:remotetea-oncrpc, which is under LGPL license.

The Ganglia integration uses gmetric4j, which is why Dropwizard Metrics removed support for Ganglia recently https://github.com/ganglia/gmetric4j/issues/9.

I'm guessing this is also an issue for Micrometer.

**Comment by shakuzen** [comment]
Thank you for bringing this to our attention. I'm not sure we have a good option other than to drop Ganglia support, then.

**Comment by shakuzen** [comment]
I have added a note to the `micrometer-registry-ganglia` module's README in d580cb7. I have also added the note to the documentation site with https://github.com/micrometer-metrics/micrometer-docs/issues/118. I am not a lawyer, but I think it is up to users of `micrometer-registry-ganglia` to determine whether what is documented prevents them from using it or not based on their situation. As far as I can tell, our usage does not force the LGPL license on Micrometer. Popular libraries like Hibernate are LGPL licensed, and we provide optional integration with them as well without being LGPL licensed ourselves.
